src/main/java/nl/ordina/elwa/fullstack/lexer/operator/Operators.java

```diff
@@ -0,0 +1,38 @@
+package nl.ordina.elwa.fullstack.lexer.operator;
+
+import static lombok.AccessLevel.PRIVATE;
+import static nl.ordina.elwa.fullstack.lexer.operator.BinaryOperator.ROOT_EXTRACTION;
+
+import java.util.Collections;
+import java.util.List;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = PRIVATE)
+public final class Operators {
+
+  /**
+   * List of known/implemented operators.
+   */
+  @SuppressWarnings("java:S1700")
```

Comment:
Is deze nog wel nodig?

Reply:
Ja, want dit gaat over de naam van de constante, die hetzelfde is als de naam van de klasse.